djstripe/settings.py

```diff
@@ -150,6 +151,23 @@ def get_subscriber_model():
     return subscriber_model
 
 
+def get_api_version():
+    """Get the desired API version to use for Stripe requests."""
+    version = getattr(settings, 'DJSTRIPE_API_VERSION', None) or '2017-02-14'
+
+    if version == 'latest':
+        version = None
+    else:
+        if not date_re.match(version):
+            raise ImproperlyConfigured(
```

Comment:
You can write a check instead (see `checks.py` for how). Runtime validation is icky.